docs/src/reference/the-traversal.asciidoc

```diff
@@ -4162,6 +4168,25 @@ In OLAP, where the atomic unit of computing is the vertex and its local "star gr
 anonymous traversals do not leave the confines of the vertex's star graph. In other words, they can not traverse to
 an adjacent vertex's properties or edges.
 
+==== Using limit(), skip(), range() inside repeat()
+
+When using globally-scoped (default scope) `limit()`, `skip()`, or `range()` steps inside `repeat()`, each iteration of 
```

Comment:
I don't think there's enough context here introducing this concept, but I'm also not sure it needs to be introduced. As this is the Reference Documentation, does it need this special description of these steps inside of `repeat`? In other words, if you just came to this section as a new user, you wouldn't have any frame of reference for how else these steps were to interact with each other. You'd only have how they work together today. I feel like it would be far more useful to have the behavior of `times` described to me clearly, with patterns or use cases that helped explain what the behavior is.